### PR TITLE
PR: Redirecting server stderr and stdout to files

### LINKED ISSUE #60: Redirect server output and errors to a file

### PR DESCRIPTION
**Pull request by andfoy**
Fixes #60

### CODE REVIEW

**Comment by andfoy** [comment]
@ccordoba12 Is this correct?

**Comment by ccordoba12** [comment]
I'll let you know tomorrow. I'm very tired now ;-)

File: spyder_terminal/terminalplugin.py [hunk 1]
```diff
@@ -57,11 +57,18 @@ def __init__(self, parent):
...
+        self.server_stdout = open(stdout_file, 'w')
+        self.server_stderr = open(stderr_file, 'w')
+
         self.server = subprocess.Popen(
             [sys.executable, osp.join(LOCATION, 'server', 'main.py'),
              '--port', str(self.port)],
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE)
+            stdout=self.server_stdout,
+            stderr=self.server_stderr)
```
Comment (ccordoba12) [inline spyder_terminal/terminalplugin.py]: Let's make the server write to a file only if `DEV` is `True`. `DEV` can be imported from `config.base` and it's `True` when you start Spyder with `bootstrap.py`.

**Comment by ccordoba12** [comment]
Looks good. Please add a test for this by setting `DEV=True` in our tests, then verifying that stdout/stderr files are written to disk.

By the way, could we use a single file for both outputs?

**Comment by andfoy** [comment]
Yes, it is possible to redirect both outputs to a single file

**Comment by ccordoba12** [comment]
Ok, but can we prefix it with something like `STDOUT: foo ...`, `STDERR: baz ...` to make it easier to read a single file?

**Comment by ccordoba12** [comment]
If it's too much trouble, then we should leave things as they are, though ;-)

**Comment by andfoy** [comment]
I think it could be possible if we implement a file wrapper around a single file, but in terms of concurrency, I don't know what would be the expected behaviour

**Comment by ccordoba12** [comment]
Then please leave two files. In any case, this feature is just for debugging purposes.

So just add a test for this and we're ready!

File: spyder_terminal/__init__.py [hunk 2]
```diff
@@ -12,5 +12,5 @@
 if PYQT5:
     from .terminalplugin import TerminalPlugin as PLUGIN_CLASS
 
-VERSION_INFO = (0, 1, 1)
+VERSION_INFO = (0, 1, 2)
```
Comment (ccordoba12) [inline spyder_terminal/__init__.py]: The dev version should include a dev0, like this

https://github.com/spyder-ide/spyder/blob/master/spyder/__init__.py#L30

**Pull request merged**